edalize/rivierapro.py

```diff
@@ -84,7 +90,20 @@ def _write_build_rtl_tcl_file(self, tcl_main):
                 args += ['-quiet']
                 args += ['-work', f.logical_name]
                 args += [f.name.replace('\\','/')]
-                tcl_build_rtl.write("{} {}\n".format(cmd, ' '.join(args)))
+                common_compilation += [f.name.replace('\\','/'),'\\\n']
```

Comment:
If Riviera uses forward slashes on all platforms, it's better to call the `get_fileset_files()` function with `force_slash=True`